code/modules/shuttle/on_move.dm

```diff
@@ -266,6 +266,16 @@ All ShuttleMove procs go here
 	if(level==1)
 		hide(T.intact)
 
+/obj/machinery/conveyor/shuttle/afterShuttleMove()
+	. = ..()
+	for(var/obj/machinery/conveyor/C in orange(1, src)) //Automatically assigned direction based on the dir of an adjacent conveyor belt to prevent desync
```

Comment:
This will also pick up diagonals and potentially fuck up with 2 different direction but parallel conveyor lines.